docs/inference.rst

```diff
@@ -0,0 +1,156 @@
+    GPS_START_TIME=$((${TRIGGER_TIME_INT} - ${SEGLEN}))
+    GPS_END_TIME=$((${TRIGGER_TIME_INT} + ${SEGLEN}))
+
+    # create injection file
+    lalapps_inspinj \
+        --output ${INJ_PATH} \
+        --seed 1000 \
+        --f-lower ${INJ_F_MIN} \
+        --waveform ${INJ_APPROX} \
+        --amp-order 7 \
+        --gps-start-time ${TRIGGER_TIME} \
+        --gps-end-time ${TRIGGER_TIME} \
+        --time-step 1 \
+        --t-distr fixed \
+        --l-distr fixed \
+        --longitude ${LONGITUDE} \
+        --latitude ${LATITUDE} \
+        --d-distr uniform \
+        --min-distance ${DISTANCE} \
+        --max-distance ${DISTANCE} \
+        --i-distr fixed \
+        --fixed-inc ${INC} \
+        --coa-phase-distr fixed \
+        --fixed-coa-phase ${COA_PHASE} \
+        --polarization ${POLARIZATION} \
+        --m-distr fixMasses \
+        --fixed-mass1 ${MASS1} \
+        --fixed-mass2 ${MASS2} \
+        --taper-injection ${TAPER} \
+        --disable-spin
+
+    # run sampler
+    pycbc_inference --verbose \
+        --instruments ${IFOS} \
+        --gps-start-time ${GPS_START_TIME} \
+        --gps-end-time ${GPS_END_TIME} \
+        --psd-model ${STRAIN} \
+        --fake-strain ${STRAIN} \
+        --sample-rate ${SAMPLE_RATE} \
+        --low-frequency-cutoff ${F_MIN} \
+        --channel-name H1:FOOBAR L1:FOOBAR \
+        --injection-file ${INJ_PATH} \
+        --processing-scheme ${PROCESSING_SCHEME} \
+        --sampler kombine \
+        --likelihood-evaluator gaussian \
+        --nwalkers ${N_WALKERS} \
+        --niterations ${N_ITERATIONS} \
+        --config-file ${CONFIG_PATH} \
+        --output-file ${OUTPUT} \
+        --checkpoint-interval ${N_CHECKPOINT} \
+        --nprocesses ${NPROCS}
+
+---------------------------------------------------
+HDF output file handler: ``pycbc.io.InferenceFile``
+---------------------------------------------------
+
+The executable ``pycbc_inference`` will write a HDF file with all the samples from each walker along with the PSDs and some meta-data about the sampler. There is a handler class ``pycbc.io.InferenceFile`` that extends ``h5py.File``. To read the output file you can do::
+
+    from pycbc.io import InferenceFile
+    fp = InferenceFile("cbc_example-n1e4.hdf.hdf", "r")
+
+To get all samples for ``mass1`` from the first walker you can do::
+
+    samples = fp.read_samples("mass1", walkers=0)
+    print samples.mass1
+
+The function ``InferenceFile.read_samples`` includes the options to thin the samples. By default the function will return samples beginning at the end of the burn-in to the last written sample, and will use the autocorrelation length (ACL) calcualted by ``pycbc_inference`` to select the indepdedent samples. You can supply ``thin_start``, ``thin_end``, and ``thin_interval`` to override this. To read all samples you would do::
+
+    samples = fp.read_samples("mass1", walkers=0, thin_start=0, thin_end=-1, thin_interval=1)
+    print samples.mass1
```

Comment:
Do you think it would be worthwhile to include a blurb here about the derived parameters? For example, something like:
```
Some standard parameters that are derived from the variable args can also be retrieved. For example, if the variable args include ``mass1`` and ``mass2``, then you can retrieve the chirp mass with::
   samples = samples = fp.read_samples("mchirp")
   print samples.mchirp

In this case, read_samples will retrieve ``mass1`` and ``mass2`` (since they are needed to compute chirp mass); ``samples.mchirp`` then returns an array of the chirp mass computed from ``mass1`` and ``mass2``. 

For more information, including the list of predefined derived parameters, see the docstring of ``pycbc.io.InferenceFile``.
```